Front End

phama2 opened the pull request.

bernep committed:
First Frame Reading from GO

Also some prep with the HTML/CSS
@scotter1995 gave me the initial PYthon version of the Go code. I just converted it to Go.

phama2 committed:
Vertical Spacing

Added 20 px vertical spacing to elements

bnidevs committed:
Update index.html

TheTimL committed:
Add font and button style in CSS

bnidevs commented:
@bernep can you add a white image placeholder in the <img> so it passes the html checker

themanofcat committed:
Add the code snippet to the correct version of main

The main.go in frontend was behind backend, this is more accurate to what's in main

bnidevs committed:
Delete progress.css.tmp.html

bnidevs commented on frontend/style/progress.css:
```diff
@@ -58,6 +59,21 @@ body{
   --secondary-color: #000000;
 }
 
+.button {
```
can you not change the styling of the button just yet, please revert this

bnidevs committed:
Update index.html

bnidevs committed:
Update index.html

bnidevs committed:
Merge branch 'main' into frontend

bnidevs committed:
Update progress.css
